pkg/azureclients/diskclient/azure_diskclient.go

```diff
@@ -76,6 +77,7 @@ func New(config *azclients.ClientConfig) *Client {
 		rateLimiterReader: rateLimiterReader,
 		rateLimiterWriter: rateLimiterWriter,
 		subscriptionID:    config.SubscriptionID,
+		cloudName:         config.CloudName,
```

Comment:
I think we already support azure stack now, see L63:
https://github.com/kubernetes-sigs/cloud-provider-azure/blob/083cf6de2feb34b98873082c60b6cd0eaefc925b/pkg/azureclients/diskclient/azure_diskclient.go#L62-L64

Reply:
then, we should remove L63, and prefer the new configure?

Reply:
then why not use `CloudName` from azure cloud config direclty?